### PR TITLE
[IMP] business_requirement: Add dates and audit info (BR000059)

### PR DESCRIPTION
**Pull request by martintorresvm**

### CODE REVIEW

File: business_requirement/views/business_view.xml [hunk 1]
```diff
@@ -61,6 +61,12 @@
                                 <group col="4">
                                     <field name="parent_id"/>
                                     <field name="level"/>
+                                    <field name="creation_date"/>
```
Comment (elicoidal) [inline business_requirement/views/business_view.xml]: You might want to have 2 columns. eg:

| Created by : Eric | on: 2016/02/20 |
| --- | --- |
| Confirmed by : Eric | on: 2016/02/20 |
| ------------------------- | ------------------------ |
| Approved by : Eric | on: 2016/02/20 |
| ------------------------- | ------------------------ |

(table is not that clean but I hope you understand the idea.)

**Comment by martintorresvm** [comment]
cc @elicoidal

File: business_requirement/models/business.py [hunk 2]
```diff
@@ -100,6 +101,39 @@ class BusinessRequirement(models.Model):
         string='Count',
         compute='_sub_br_count'
     )
+    creation_date = fields.Datetime(
```
Comment (duanyp1991) [inline business_requirement/models/business.py]: This field has already exist.

File: business_requirement/models/business.py [hunk 3]
```diff
@@ -100,6 +101,39 @@ class BusinessRequirement(models.Model):
...
+        string='Creation Date',
+        default=fields.Datetime.now,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    created_by = fields.Many2one(
```
Comment (duanyp1991) [inline business_requirement/models/business.py]: This field has already exist.

**Comment by elicoidal** [comment]
@victormartinelicocorp is it the last one, including arrangement you sent me yesterday?
Except for @duanyp1991 remarks, LGTM good enough anyway for the moment

File: business_requirement/models/business.py [hunk 4]
```diff
@@ -100,6 +101,39 @@ class BusinessRequirement(models.Model):
...
+        readonly=True,
+        default=lambda self: self.env.user,
+        states={'draft': [('readonly', False)]}
+    )
+    confirmation_date = fields.Datetime(
+        string='Confirmation Date',
+        readonly=True,
+        states={'confirmed': [('readonly', False)]}
+    )
+    confirmed_by = fields.Many2one(
```
Comment (duanyp1991) [inline business_requirement/models/business.py]: change the name with `_id`

**Comment by martintorresvm** [comment]
@elicoidal Yes it is. I have to correct the points about @duanyp1991 comments.

cc @elicoidal

**Comment by martintorresvm** [comment]
cc @duanyp1991

**Comment by duanyp1991** [comment]
LGTM

**Pull request merged**